ocaml/idl/ocaml_backend/gen_api.ml

```diff
@@ -72,6 +72,23 @@ let overrides =
     )
   ]
 
+(** Generate enum__all and enum_to_string bindings for all enums *)
```

Comment:
Is there a reason why we use two underscores in `x__all`?

Reply:
It's out of caution, I don't want this new method to create current or future name collisions